CaT/test/Filter/SqlInterpreterTest.php

```diff
@@ -0,0 +1,407 @@
+<?php
+
+/* Copyright (c) 2015 Richard Klees, Extended GPL, see docs/LICENSE */
+
+class  SqlInterpreterTest extends PHPUnit_Framework_TestCase {
+	protected $backupGlobals = FALSE;
+
+	protected function setUp() {
+		PHPUnit_Framework_Error_Deprecated::$enabled = FALSE;
+
+		include_once("./Services/PHPUnit/classes/class.ilUnitUtil.php");
+		ilUnitUtil::performInitialisation();
+		global $ilDB;
+		$this->db = $ilDB;
+		$this->factory = new \CaT\Filter\PredicateFactory();
+		$this->interpreter = new \CaT\Filter\SQLPredicateInterpreter($ilDB);
+		date_default_timezone_set('Europe/Berlin');
+	}
+
+	public function test_TRUE() {
+		$f = $this->factory;
+		$i = $this->interpreter;
+
+		$p = $f->_TRUE();
+
+		$this->assertEquals($i->interpret($p) , 'TRUE ');
```

Comment:
Bitte nur interessenhalber einmal abklären: Gibt es TRUE auch in sqlite, postgres, mariaDB, oracleDB? Ergebniss bitte per Mail an mich, danke!